dist/tests/inverted-index-test.js

```diff
@@ -1,14 +1,82 @@
 'use strict';
 
+var _supertest = require('supertest');
+
+var _supertest2 = _interopRequireDefault(_supertest);
+
+var _path = require('path');
+
+var _path2 = _interopRequireDefault(_path);
+
 var _invertedIndex = require('../src/inverted-index');
 
 var _invertedIndex2 = _interopRequireDefault(_invertedIndex);
 
+var _server = require('../server');
+
+var _server2 = _interopRequireDefault(_server);
+
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 _invertedIndex2.default.createIndex('book1.json', _invertedIndex2.default.readFile('book1.json'));
 _invertedIndex2.default.createIndex('book2.json', _invertedIndex2.default.readFile('book2.json'));
-_invertedIndex2.default.createIndex('book2.json', _invertedIndex2.default.readFile('invalidBook.json'));
+
+var createdIndexForBook1 = {
+  'book1.json': {
```

Comment:
It would be better if you could move all these into an external file and then import the variables.